cooltools/lib/common.py

```diff
@@ -9,6 +9,70 @@
 import bioframe
 
 
+def assign_view_2D(
+    features,
+    view_df,
+    cols=["chrom1", "start1", "end1", "chrom2", "start2", "end2"],
```

Comment:
split into `cols1` and `cols2` or `_left` `_right`

or keep them together but call differently `cols_paired`